cli/alert.go

```diff
@@ -71,6 +71,7 @@ func configureAlertCmd(app *kingpin.Application) {
 	queryCmd.Flag("silenced", "Show silenced alerts").Short('s').BoolVar(&a.silenced)
 	queryCmd.Flag("active", "Show active alerts").Short('a').BoolVar(&a.active)
 	queryCmd.Flag("unprocessed", "Show unprocessed alerts").Short('u').BoolVar(&a.unprocessed)
+	queryCmd.Flag("receiver", "Show alerts matching receiver").Short('r').StringVar(&a.receiver)
```

Comment:
Can you add to `Supports regex syntax`? `amtool alert query -r 'backend-.*'` for example is a useful query.